### PR TITLE
add docs for tidb binlog

### PR DESCRIPTION
**Pull request by WangXiangUSTC**
### What problem does this PR solve? 
add documents for tidb-binlog
user guide
monitor doc
slave client user guide
problem feedback
issue template

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/tidb-binlog-support.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+---
+title: tidb-binlog support
+category: tools
+---
+
+# TiDB-Binlog Support
+
+Before create a issue, please read this document first:
+* [问题排查及反馈流程](../tidb-binlog-issue.md)
```
Comment (IANTHEREAL) [inline .github/ISSUE_TEMPLATE/tidb-binlog-support.md]: I'm afraid user would't read it

File: .github/ISSUE_TEMPLATE/tidb-binlog-support.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
+---
+title: tidb-binlog support
+category: tools
+---
+
+# TiDB-Binlog Support
+
+Before create a issue, please read this document first:
```
Comment (IANTHEREAL) [inline .github/ISSUE_TEMPLATE/tidb-binlog-support.md:8]: lightning issue template is a good example.  you can learn from it @WangXiangUSTC

**Comment by IANTHEREAL** [comment]
nice guy

File: docs/tidb-binlog/zh_CN/tidb-binlog-issue.md [hunk 3]
```diff
@@ -0,0 +1,90 @@
...
+
+## 检查 TiDB-Binlog 运行情况
+
+在上报问题前，请根据 checklist 列出的事项对 binlog 的运行情况进行检查。
+
+### kafka 版本
+* TiDB 是否开启了 Binlog
+    * 查看 TiDB 启动脚本及配置，或者直接查看 TiDB 的日志中记录的配置参数，开启 binlog 的命令行参数为 binlog-socket，在配置文件中为 [binlog] 中的 socket。
+* 查看 Pump、Drainer 的监控
+    * 是否有指标异常。
```
Comment (IANTHEREAL) [inline docs/tidb-binlog/zh_CN/tidb-binlog-issue.md]: we should add docuement how to determine the metrics anomaly， please mark it in jira issue

File: docs/tidb-binlog/zh_CN/tidb-binlog-issue.md [hunk 4]
```diff
@@ -0,0 +1,90 @@
...
+* 查看所有的 Pump、Drainer 状态是否正常
+    * 使用 binlogctl 工具查看，主要关注状态是否为 online，Pump 的 kafka offset 的位置是否正常（大于0），以及有没有之前没正常下线的 Pump/Drainer。
```
Comment (IANTHEREAL) [inline docs/tidb-binlog/zh_CN/tidb-binlog-issue.md]: too vague， you need a checklist

**Comment by WangXiangUSTC** [comment]
@GregoryIan  PTAL again

File: .github/ISSUE_TEMPLATE/tidb-binlog-support.md [hunk 5]
```diff
@@ -0,0 +1,46 @@
+---
+title: tidb-binlog support
+category: tools
+---
+
+## TiDB-Binlog Support
+
+Before create a issue, please read this document first:
+* [问题排查及反馈流程](../../docs/tidb-binlog/zh_CN/tidb-binlog-issue.md)
```
Comment (IANTHEREAL) [inline .github/ISSUE_TEMPLATE/tidb-binlog-support.md]: zh_cn:

File: docs/tidb-binlog/zh_CN/README.md [hunk 6]
```diff
@@ -0,0 +1,18 @@
+# TiDB-Binlog
+
+TiDB-Binlog 是一个用于收集 TiDB 的 Binlog，并提供实时备份和同步功能的商业工具。
+
+TiDB-Binlog 支持以下功能场景:
+
+* **数据同步**：同步 TiDB 集群数据到其他数据库
+* **实时备份和恢复**：备份 TiDB 集群数据，同时可以用于 TiDB 集群故障时恢复
+
```
Comment (IANTHEREAL) [inline docs/tidb-binlog/zh_CN/README.md]: add index to kafka and local document

**Comment by IANTHEREAL** [comment]
@july2993 PTAL

**Comment by IANTHEREAL** [comment]
please create a jira task (en_US document) and paste here

**Comment by WangXiangUSTC** [comment]
jira issue: https://internal.pingcap.net/jira/browse/TOOL-541

**Comment by WangXiangUSTC** [comment]
@GregoryIan @july2993 PTAL again

**Comment by IANTHEREAL** [comment]
add `check whether table has primaey key or unique index` and `whether user load data into tidb`

**Comment by IANTHEREAL** [comment]
ping @WangXiangUSTC

**Comment by IANTHEREAL** [comment]
LGTM,  @july2993  PTAL

**Comment by july2993** [comment]
LGTM

**Pull request merged**